src/common/datetime.cpp

```diff
@@ -541,6 +541,52 @@ bool wxDateTime::IsLeapYear(int year, wxDateTime::Calendar cal)
     }
 }
 
+#ifdef _WIN32
+#include "wx/msw/registry.h"
+
+/* static */
+bool wxDateTime::GetFirstWeekDay(wxDateTime::WeekDay *firstDay)
+{
+    wxRegKey key(wxRegKey::HKCU, "Control Panel\\International");
```

Comment:
I think there should be

    wxCHECK_MSG( firstDay, false, wxS("output parameter must be non-null") );

here (and, annoyingly, in the implementations of this function for the other platforms... until we want to split it into a public function and platform-dependent `DoGetFirstWeekDay()` which looks like an overkill).